src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/CallbackVersion.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks;
+
+public enum CallbackVersion {
+    V_1,
```

Comment:
not sure we are/will use this

Reply:
I know but for completeness I would like to leave it in. It would be confusing if its not here.